### PR TITLE
Add 'lfp' report type and 'electrodes_file' field

### PR DESCRIPTION
**Pull request by jorblancoa**
I also updated the documentation here:
https://github.com/BlueBrain/sonata-extension/pull/11

### CODE REVIEW

File: src/config.cpp [hunk 1]
```diff
@@ -868,6 +869,7 @@ class SimulationConfig::Parser
         parseOptional(*runIt, "ionchannel_seed", result.ionchannelSeed, {0});
         parseOptional(*runIt, "minis_seed", result.minisSeed, {0});
         parseOptional(*runIt, "synapse_seed", result.synapseSeed, {0});
+        parseOptional(*runIt, "electrodes_file", result.electrodesFile, {""});
```
Comment (WeinaJi) [inline src/config.cpp:872]: I think it is safer to parse it to be an absolute path here, like the other file paths in the config file.

Comment (jorblancoa) [reply src/config.cpp:872]: The optional files are parsed in the same way. For example:

`parseOptional(*outputIt, "log_file", result.logFile, {""});
 parseOptional(*outputIt, "spikes_file", result.spikesFile, {"out.h5"});`

**Review by WeinaJi — APPROVED** [review]
LGTM. Let's wait to merge the SPEC first https://github.com/BlueBrain/sonata-extension/pull/11

**Comment by jorblancoa** [comment]
Now that the spec has been merged, maybe we can also merge this? @mgeplf

**Pull request merged**